apps/web/components/ui/ImageText/ImageText.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div v-if="showComponent" :class="['flex flex-col md:flex-row items-center', positionClass]">
+    <div :class="['w-full', 'md:w-1/2']">
```

Comment:
```suggestion
    <div :class="['md:w-1/2']">
```
My understanding is that, unlike the banner, the image-text shouldn't be full-width.